### DISCUSSION TITLE
Plotting issue

### DISCUSSION THREAD

**Discussion by allenmoench** [earthlab-education/Earth-Analytics-AY24#648] [Q&A]
Hello! I've been trying to plot the DEM raster, with the boundary for the Pawnee National Grassland on top of it. I was able to plot the DEM just fine, and then when I started adding in code to plot the boundary on top of it, this happened:
![image](https://github.com/user-attachments/assets/c239b418-8a26-42b3-89c0-23fcb0e218ea)
Now it just prints a blank plot, with the axis in the middle. I've gone through the code with a fine-toothed comb, and can't figure out why it's doing this.

**Comment by hanried** [comment]
Is it something to do with your "pawnee_gdf" variable? That variable has a squiggly underline in your code so that makes me wonder if it's something with that.
